src/main.rs

```diff
@@ -95,7 +95,7 @@ async fn main() -> Result<()> {
         homepage: "atos.net/de/deutschland/sc".into(),
     });
 
-    env_logger::try_init()?;
+    tracing_subscriber::fmt::init();
```

Comment:
I just noticed that `tracing_subscriber::fmt::try_init()?` would be closer to the old implementation but IMO/AFAIK both variants should be fine (`init` is just `try_init().expect("Unable to install global subscriber")` (https://docs.rs/tracing-subscriber/0.3.16/src/tracing_subscriber/fmt/mod.rs.html#1231-1233) so the result is more or less the same).